bin/serve.js

```diff
@@ -46,7 +58,9 @@ var app = jserve({
 		info: console.log.bind(console)
 	},
 	middleware: [
+	interceptIndex,
```

Comment:
try an `npm run lint` in the root dir 😉 apologies for our draconian linting rules!

Reply:
Just fixed this! :)